https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/batch/pull-kubernetes-unit/1902474334799138816

```
k8s.io/kubernetes/test/e2e/framework/internal/unittests: list-labels
{Failed  === RUN   TestListLabels
    listlabels_test.go:34: 
        	Error Trace:	/home/prow/go/src/k8s.io/kubernetes/test/e2e/framework/internal/unittests/list-labels/listlabels_test.go:34
        	Error:      	Not equal: 
        	            	expected: "The following labels can be used with 'ginkgo run --label-filter':\n    Alpha\n    Beta\n    BetaOffByDefault\n    Conformance\n    Disruptive\n    Environment:Linux\n    Environment:no-such-env\n    Feature:OffByDefault\n    Feature:feature-foo\n    Feature:no-such-feature\n    FeatureGate:TestAlphaFeature\n    FeatureGate:TestBetaFeature\n    FeatureGate:TestGAFeature\n    FeatureGate:no-such-feature-gate\n    NodeConformance\n    Serial\n    Slow\n    bar\n    custom-label\n    foo\n    sig-testing\n\n"
        	            	actual  : "The following labels can be used with 'ginkgo run --label-filter':\n    Alpha\n    Beta\n    Conformance\n    Disruptive\n    Environment:Linux\n    Environment:no-such-env\n    Feature:OffByDefault\n    Feature:feature-foo\n    Feature:no-such-feature\n    FeatureGate:TestAlphaFeature\n    FeatureGate:TestBetaFeature\n    FeatureGate:TestGAFeature\n    FeatureGate:no-such-feature-gate\n    NodeConformance\n    Serial\n    Slow\n    bar\n    custom-label\n    foo\n    sig-testing\n\n"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -3,3 +3,2 @@
        	            	     Beta
        	            	-    BetaOffByDefault
        	            	     Conformance
        	Test:       	TestListLabels
--- FAIL: TestListLabels (0.00s)
}
```

I think maybe due to https://github.com/kubernetes/kubernetes/pull/130121

But we should still have an off-by-default test case 😕

--- a/test/e2e/framework/internal/unittests/bugs/bugs.go
+++ b/test/e2e/framework/internal/unittests/bugs/bugs.go
@@ -80,6 +80,7 @@ func Describe() {
 		framework.WithFeatureGate("no-such-feature-gate"),
 		framework.WithFeatureGate(features.Alpha),
 		framework.WithFeatureGate(features.Beta),
+		framework.WithFeatureGate(features.BetaDefaultOff),
 		framework.WithFeatureGate(features.GA),
 		framework.WithConformance(),
 		framework.WithNodeConformance(),
@@ -116,14 +117,14 @@ ERROR: bugs.go:71: trailing or leading spaces are unnecessary and need to be rem
 ERROR: bugs.go:76: WithFeature: unknown feature "no-such-feature"
 ERROR: bugs.go:78: WithEnvironment: unknown environment "no-such-env"
 ERROR: bugs.go:80: WithFeatureGate: the feature gate "no-such-feature-gate" is unknown
-ERROR: bugs.go:106: SIG label must be lowercase, no spaces and no sig- prefix, got instead: "123"
+ERROR: bugs.go:107: SIG label must be lowercase, no spaces and no sig- prefix, got instead: "123"
 ERROR: buggy/buggy.go:100: hello world
 ERROR: some/relative/path/buggy.go:200: with spaces
 `
 	// Used by unittests/list-tests. It's sorted by test name, not source code location.
 	ListTestsOutput = `The following spec names can be used with 'ginkgo run --focus/skip':
-    ../bugs/bugs.go:100: [sig-testing] abc   space1 space2  [Feature:no-such-feature] [Feature:feature-foo] [Environment:no-such-env] [Environment:Linux] [FeatureGate:no-such-feature-gate] [Feature:OffByDefault] [FeatureGate:TestAlphaFeature] [Alpha] [Feature:OffByDefault] [FeatureGate:TestBetaFeature] [Beta] [FeatureGate:TestGAFeature] [Conformance] [NodeConformance] [Slow] [Serial] [Disruptive] [custom-label] xyz x [foo] should [bar]
-    ../bugs/bugs.go:95: [sig-testing] abc   space1 space2  [Feature:no-such-feature] [Feature:feature-foo] [Environment:no-such-env] [Environment:Linux] [FeatureGate:no-such-feature-gate] [Feature:OffByDefault] [FeatureGate:TestAlphaFeature] [Alpha] [Feature:OffByDefault] [FeatureGate:TestBetaFeature] [Beta] [FeatureGate:TestGAFeature] [Conformance] [NodeConformance] [Slow] [Serial] [Disruptive] [custom-label] xyz y [foo] should [bar]
+    ../bugs/bugs.go:101: [sig-testing] abc   space1 space2  [Feature:no-such-feature] [Feature:feature-foo] [Environment:no-such-env] [Environment:Linux] [FeatureGate:no-such-feature-gate] [Feature:OffByDefault] [FeatureGate:TestAlphaFeature] [Alpha] [Feature:OffByDefault] [FeatureGate:TestBetaFeature] [Beta] [FeatureGate:TestBetaDefaultOffFeature] [Beta] [Feature:OffByDefault] [FeatureGate:TestGAFeature] [Conformance] [NodeConformance] [Slow] [Serial] [Disruptive] [custom-label] xyz x [foo] should [bar]
+    ../bugs/bugs.go:96: [sig-testing] abc   space1 space2  [Feature:no-such-feature] [Feature:feature-foo] [Environment:no-such-env] [Environment:Linux] [FeatureGate:no-such-feature-gate] [Feature:OffByDefault] [FeatureGate:TestAlphaFeature] [Alpha] [Feature:OffByDefault] [FeatureGate:TestBetaFeature] [Beta] [FeatureGate:TestBetaDefaultOffFeature] [Beta] [Feature:OffByDefault] [FeatureGate:TestGAFeature] [Conformance] [NodeConformance] [Slow] [Serial] [Disruptive] [custom-label] xyz y [foo] should [bar]
 
 `
 
@@ -140,6 +141,7 @@ ERROR: some/relative/path/buggy.go:200: with spaces
     Feature:feature-foo
     Feature:no-such-feature
     FeatureGate:TestAlphaFeature
+    FeatureGate:TestBetaDefaultOffFeature
     FeatureGate:TestBetaFeature
     FeatureGate:TestGAFeature
     FeatureGate:no-such-feature-gate

--- a/test/e2e/framework/internal/unittests/bugs/features/features.go
+++ b/test/e2e/framework/internal/unittests/bugs/features/features.go
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	Alpha featuregate.Feature = "TestAlphaFeature"
-	Beta  featuregate.Feature = "TestBetaFeature"
-	GA    featuregate.Feature = "TestGAFeature"
+	Alpha          featuregate.Feature = "TestAlphaFeature"
+	Beta           featuregate.Feature = "TestBetaFeature"
+	BetaDefaultOff featuregate.Feature = "TestBetaDefaultOffFeature"
+	GA             featuregate.Feature = "TestGAFeature"
 )
 
 func init() {
@@ -40,6 +41,9 @@ var testFeatureGates = map[featuregate.Feature]featuregate.VersionedSpecs{
 	Beta: {
 		{Version: version.MustParse("1.27"), Default: false, PreRelease: featuregate.Alpha},
 		{Version: version.MustParse("1.28"), Default: true, PreRelease: featuregate.Beta},
+	},
+	BetaDefaultOff: {
+		{Version: version.MustParse("1.28"), Default: false, PreRelease: featuregate.Beta},
 	},
 	GA: {
 		{Version: version.MustParse("1.27"), Default: false, PreRelease: featuregate.Alpha},

